mp/tests/proxy_counts.py

```diff
@@ -0,0 +1,118 @@
+import json
```

Comment:
This whole file was added in to test the proxy results from the marine debris database.